packages/mdc-dialog/test/component.test.ts

```diff
@@ -591,4 +591,15 @@ describe('MDCDialog', () => {
     component.layout();
     expect((component as any).foundation.layout).toHaveBeenCalled();
   });
+
+  it(`${strings.INITIAL_FOCUS_ATTRIBUTE} will focus when the dialog is opened`, () => {
+    const {component: component1, yesButton: yesButton1} = setupTest();
+    const {component: component2, yesButton: yesButton2} = setupTest();
+
+    const initialFocusEl1 = (component1.getDefaultFoundation() as any).adapter.getInitialFocusEl();
+    expect(initialFocusEl1).toEqual(yesButton1);
+
+    const initialFocusEl2 = (component2.getDefaultFoundation() as any).adapter.getInitialFocusEl();
+    expect(initialFocusEl2).toEqual(yesButton2);
```

Comment:
Let's add a more complete test:
- Add both components to document body
- Call component1.open()
- Assert that yesButton1 === document.activeElement
- Repeat for component2
- Remove both components from document body